diesel_migrations/migrations_internals/src/migration.rs

```diff
@@ -12,6 +13,31 @@ pub trait Migration {
     }
 }
 
+#[allow(missing_debug_implementations)]
+pub struct MigrationName<'a> {
```

Comment:
This type should still implement `Copy`. ;) (I'm still really surprised that the `missing_copy_implementations` lint isn't triggering here)